src/workloads/execution/SSBColumnStoreIndex.yml

```diff
@@ -0,0 +1,542 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload compares the query performance of SSB (star schema benchmark) queries with columnar indexes to performances without them.
+
+  The workload consists of the following phases and actors:
+    1. Run SSB queries with columnstore indexes.
+    2. Drop columnstore indexes.
+    3. Run SSB queries without columnstore indexes.
```

Comment:
So this workload is comparing column store indexes against no indexes (colscans). Is that a fair comparison? Should we also run the queries against regular indexes?

Reply:
Yes, we want to see the speed-up (or penalty) compared to collection scans. In particular we want to evaluate CSI in contexts where the collection does not fit into memory.